index.html

```diff
@@ -5696,7 +5696,7 @@ <h3 id='extensions'>Extensions</h3>
 </tr>
 <tr>
 <td>item.item-status</td>
-<td>Indicates whether the Charge line is presently active or cancelled. If a Charge is deleted, voided, or voiding it will have a status of Cancelled. If a closed charge is recreated through a correction, the original Charge will be voided with a status of Cancelled and a new Charge will be created to replace it with an Active status.</td>
+<td>Indicates current ledger entry status.</td>
```

Comment:
1:56 AM
Ryan Gonet <td>Indicates current ledger entry status.</td> Can you elaborate on this more please? It should be super clear that this has literally nothing to do with the claim itself.

Reply:
Erin Norris:eyeroll:  2 hours ago
I dont know what else to say.  iff you want to provide text feel free.

Reply:
Fixed.